app/api/payments/paypal/capture/route.ts

```
@@ -1,85 +1,95 @@
       }
 
-      // Send notification to user
       const { data: transaction } = await supabase
         .from("transactions")
         .select("user_id, amount")
         .eq("reference", orderId)
-        .single()
+        .single();
 
-      if (transaction) {
+      if (transaction && transaction.user_id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)